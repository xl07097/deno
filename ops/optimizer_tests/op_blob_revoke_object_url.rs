pub fn op_blob_revoke_object_url(
  state: &mut OpState,
  url: String,
) -> Result<(), AnyError> {
  // TODO(@littledivy): fast compatible https://github.com/denoland/deno/issues/17159
  let url = Url::parse(&url)?;
  let blob_store = state.borrow::<BlobStore>();
  blob_store.remove_object_url(&url);
  Ok(())
}
