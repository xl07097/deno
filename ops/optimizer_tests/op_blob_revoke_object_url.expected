=== Optimizer Dump ===
returns_result: true
has_ref_opstate: true
has_rc_opstate: false
has_fast_callback_option: false
needs_fast_callback_option: true
fast_result: Some(Void)
fast_parameters: [V8Value, SeqOneByteString]
transforms: {1: Transform { kind: SeqOneByteString(Owned), index: 1 }}
is_async: false
fast_compatible: false
